client/src/modules/journal/journal.js

```diff
@@ -312,14 +322,20 @@ function JournalController(Journal, Sorting, Grouping,
   function load(options) {
     vm.loading = true;
     vm.hasError = false;
-
+    var transactionNumber = 0;
+    vm.gridOptions.gridFooterTemplate = null;
+    vm.gridOptions.showGridFooter = false;
     // @fixme
     Journal.grid(null, options)
       .then(function (records) {
+        // To Get the number of transaction 
+        transactionNumber =  records.aggregate.length;
 
         // pre process data - this should be done in a more generic way in a service
         vm.gridOptions.data = transactions.preprocessJournalData(records);
-
+        vm.gridOptions.showGridFooter = true;
+        vm.gridOptions.gridFooterTemplate = '<div><strong>' + $translate.instant('FORM.INFO.NUM_TRANSACTION') + 
+          ' : ' + transactionNumber + ' / ' + vm.numberTransaction + '</strong></div>'; 
```

Comment:
@lomamech can you rename these variables to something that provides more context? There is no real difference between `numberTransaction` and `transactionNumber` - this might be hard to follow when revisiting this code in the future. 

For example: 
```js
// number of all of the transactions in the 
var numberTotalSystemTransactions

// number of transactions downloaded and shown in the current journal 
var numberCurrentGridTransactions
```